tests_integration/spcs/testing_utils/compute_pool_utils.py

```diff
@@ -57,7 +57,7 @@ def create_compute_pool(self, compute_pool_name: str) -> None:
         assert result.json, result.output
         assert "status" in result.json
         assert (
-            f"Compute Pool {compute_pool_name.upper()} successfully created."
+            f"Compute pool {compute_pool_name.upper()} successfully created."
```

Comment:
What do you think about a case-insensitive comparison? I'm thinking of, in the short-term, deployments that might not have this BCR.

Reply:
Yeah I can do that, but can I merge this in first to unblock other PRs?